examples/kubernetes/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: example-hotrod
+  name: example-hotrod
+  namespace: example-hotrod
+spec:
```

Comment:
I don't see this running the actual Jaeger backend.

Reply:
The use-case I have in mind for this is that you already have some tracing backend working and you need to test it out with an example app. I could make a different config that also deploys the all-in-one Jaeger alongside the app.

Reply:
I love the idea of a k8s example, but it really needs to start up the jaeger as well. That way people just getting into Jaeger can easily spin up an example app and functioning backend and see the basic configuration points to get things working.

Reply:
I agree. Added in https://github.com/jaegertracing/jaeger/pull/3645/commits/12122dd398c40fa59f9fd23e0710732c1b61c4c3 however I'm not familiar with how the all-in-one is supposed to work. I have it running but the traces don't seem to be showing up. If I have some time later I'll poke around with it a bit more.

Reply:
Oh! I think I got it. The port needs to be `UDP` not `TCP`. Testing that out now.

Reply:
Nope, that doesn't seem to be it. Will try again later.

Reply:
Got it working by moving the jaeger all-in-one container in to the same pod and using localhost as the jaeger URL. 🎉